loadgen/bindings/c_api.h

```diff
@@ -42,6 +41,10 @@ typedef void (*FlushQueriesCallback)();
 typedef void (*ReportLatencyResultsCallback)(ClientData, const int64_t*,
                                              size_t);
 
+/// \breif SUT calls this function to report query result back to loadgen
```

Comment:
Spelled "brief"

Reply:
thanks for the review, somehow I messed up the change after resubmit. I will start a new merge request with the new change